variants/vmware_template.ovf

```diff
@@ -0,0 +1,113 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<Envelope xmlns="http://schemas.dmtf.org/ovf/envelope/1" xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1" xmlns:vmw="http://www.vmware.com/schema/ovf" xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData" xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData">
+  <References>
+    <File ovf:id="file1" ovf:href="{{ROOT_DISK}}"/>
+    <File ovf:id="file2" ovf:href="{{DATA_DISK}}"/>
+  </References>
+  <DiskSection>
+    <Info>List of the virtual disks</Info>
+    <Disk ovf:capacityAllocationUnits="byte" ovf:format="http://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized" ovf:diskId="vmdisk1" ovf:capacity="2147483648" ovf:fileRef="file1"/>
+    <Disk ovf:capacityAllocationUnits="byte" ovf:format="http://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized" ovf:diskId="vmdisk2" ovf:capacity="21474836480" ovf:fileRef="file2"/>
+  </DiskSection>
+  <NetworkSection>
+    <Info>The list of logical networks</Info>
+    <Network ovf:name="VM Network">
+      <Description>The network</Description>
+    </Network>
+  </NetworkSection>
+  <vmw:StorageGroupSection ovf:required="false" vmw:id="group1" vmw:name="VMC Workload Storage Policy - Cluster-1">
+    <Info>Storage policy for group of disks</Info>
+    <vmw:Description>The VMC Workload Storage Policy - Cluster-1 storage policy group</vmw:Description>
+  </vmw:StorageGroupSection>
+  <VirtualSystem ovf:id="Bottlerocket-{{VERSION}}">
+    <Info>A Virtual system</Info>
+    <Name>Bottlerocket-{{VERSION}}</Name>
```

Comment:
Where do these identifiers show up?

Overall I'd like to align the presentation with the way AMIs are named, and with fields in `/etc/os-release`.

Reply:
Removed the `Name` as it's not necessary and the [spec](https://www.dmtf.org/sites/default/files/standards/documents/DSP0243_1.1.0.pdf) (line 575) makes me think it's really only used inside the OVF.  It doesn't show up anywhere in vSphere that I saw.